web/html/src/branding/css/susemanager/variables.less

```diff
@@ -108,6 +108,7 @@
 @link-color:             @eos-bc-blue-500;
 // TODO: This should be selection green instead?
 @table-row-selected:     lighten(@eos-bc-yellow-100, 5%);
+@badge-bg: @gray-dark;
```

Comment:
I would use `@eos-bc-gray-1000` here directly. Other than that looks good. This fixes the issue in the Suma theme, is the Uyuni theme not affected?

Reply:
Thank you for the review. I changed the variable as you suggested and applied the change to Uyuni theme, could you review it again?